crates/rolldown/src/types/bundle_output.rs

```diff
@@ -6,4 +7,5 @@ pub struct BundleOutput {
   pub warnings: Vec<BuildDiagnostic>,
   pub errors: Vec<BuildDiagnostic>,
   pub assets: Vec<Output>,
+  pub watch_files: Vec<ArcStr>,
```

Comment:
The rollup also used it to store files which need to watched, ref https://github.com/rollup/rollup/blob/master/src/watch/watch.ts#L248